__tests__/lint/assertions-property-key-not-allowed/snapshot.js

```diff
@@ -0,0 +1,11 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`E2E lint assertions-property-key-not-allowed 1`] = `
+
+validating /openapi.yaml...
+Something went wrong when processing /Users/andrewtatomyr/Tests/redocly-cli-backup/__tests__/lint/assertions-property-key-not-allowed/openapi.yaml:
+
+  - The 'disallowed' assertion can't be used on properties. Please remove the 'property' key.
```

Comment:
Currently the error is:

`  - disallowed can't be used on a single property. Please use 'property'..`